splinterd/src/main.rs

```diff
@@ -538,7 +577,7 @@ fn get_log_filter_level(matches: &ArgMatches) -> log::LevelFilter {
     }
 }
 
-fn start_daemon(matches: ArgMatches) -> Result<(), UserError> {
+fn start_daemon(matches: ArgMatches, _log_handle: Handle) -> Result<(), UserError> {
```

Comment:
Why the unused param?

Reply:
Its only used with experimental turned on and throws a warning otherwise